Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and '$(RUNTIME_PATH)' != ''">%RUNTIME_PATH%\</TestRuntimeEnvVar>
```

Comment:
These environment variables are never passed through MSBuild itself; they are:
* Passed in to RunTests.cmd (`RUNTIME_PATH`)
* Inferred from the RunTests scripts location (`EXECUTION_DIR`).

You should instead just change the `StartProgram`, `StartArguments` stuff down below so that this change only affects VS.